include/Probe.php

```diff
@@ -57,6 +59,8 @@ private function xrd($host) {
 		$ssl_url = "https://".$host."/.well-known/host-meta";
 		$url = "http://".$host."/.well-known/host-meta";
 
+		$baseurl = "http://".$host;
```

Comment:
I want to set it only if it hasn't failed. So it has to be set **after* the last "return false".